app/collections/[slug]/page.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
```

Comment:
There's currently an `app/collections/lane/[slug]/page.tsx` file. What's the difference between that route and this route?